### PR TITLE
chore: port CLI docgen

### PR DESCRIPTION
**Pull request by WitoDelnat**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/cli/testkube_abort.md [hunk 1]
```diff
@@ -12,18 +12,6 @@ testkube abort <resourceName> [flags]
   -h, --help   help for abort
 ```
 
-### Options inherited from parent commands
```
Comment (WitoDelnat) [inline docs/cli/testkube_abort.md:15]: I'm not sure why it suddenly hides these but I don't want to lose a lot of time to investigate. I guess this is alright for now?

Comment (olensmar) [reply docs/cli/testkube_abort.md:15]: let's create a follow-up ticket for this though - doesn't seem right that all inherited params are suddenly gone..

**Review by olensmar — APPROVED** [review]

**Comment by olensmar** [comment]
@WitoDelnat @vsukhin is this PR still applicable!? It seems like the CLI docs have been updated separately..

**Comment by WitoDelnat** [comment]
Yeah not sure what to do with this PR. I'll close it for now and we can always re-open it if it was relevant after all.

**Pull request closed without merging**